core/version/src/lib.rs

```diff
@@ -63,25 +74,38 @@ pub mod notifier {
                 ),
                 Ok(releases) => {
                     for r in releases.into_iter() {
+                        counter!("version.new", 1);
                         release_dao
                             .new_release(r)
                             .await
-                            .map_err(|e| log::debug!("Problem storing new release. {}", e))
+                            .map_err(|e| log::error!("Storing new Yagna release to DB. {}", e))
                             .ok();
                     }
                 }
             };
         }
     }
 
-    pub async fn pinger(db: DbExecutor) {
-        let mut interval = time::interval(Duration::from_secs(60 * 24));
-        let release_dao = db.as_dao::<crate::db::dao::ReleaseDAO>();
+    pub(crate) async fn pinger(db: DbExecutor) -> ! {
+        // TODO: make interval configurable
+        // TODO: after test make it 30min instead 30sec
+        let mut interval = tokio::time::interval(Duration::from_secs(30));
         loop {
+            let release_dao = db.as_dao::<ReleaseDAO>();
             interval.tick().await;
-            if let Ok(Some(db_release)) = release_dao.pending_release().await {
-                log::warn!("New version of yagna available {}! Close yagna and run `{}` to install or `{}` to mute this notification", db_release, UPDATE_CURL, SILENCE_CMD);
-            };
+            match release_dao.pending_release().await {
+                Ok(Some(release)) => {
+                    if !release.seen {
+                        log::warn!(
+                            "New Yagna version {} ({}) is available. TODO: add howto here",
+                            release.name,
+                            release.version
+                        )
```

Comment:
Why not use `Display` of `Release` as before?

Reply:
I'm using `thiserror` to implement ToString and Display for the release.